src/main/java/ch/njol/skript/conditions/CondIsNamed.java

```diff
@@ -59,6 +56,7 @@ public class CondIsNamed extends Condition {
 	@Nullable
 	private Expression<String> name;
 	private Expression<Object> objects;
+	private boolean caseSensitive = false;
```

Comment:
I might be able to, but I'm not 100% sure if that takes in account for changing it and doing `/sk reload config`
since the setting it in `init` has been something even `effReplace` does

Reply:
It can be static final so it'll update all the references at once when reloading